frontend/src/pages/auth/signin/SignIn.tsx

```diff
@@ -41,13 +42,14 @@ const SignIn: React.FC = () => {
     const handleSignIn = async () => {
         if (!isValid) return;
         const token = await logInService(authCredentials);
-        const decodedToken = decodeToken(token);
+        if (token === "") return;
```

Comment:
Actually if token is empty , smth goes wrong and you have to throw an error. But you intercept errors, so I don't think you have to check it